### ISSUE TITLE
add launch to heroku button

### ISSUE THREAD

**Issue by ashwin31** [MicroPyramid/Django-CRM#1]

**Comment by inoks** [comment]
Why this was closed? Do you want a PR?

**Comment by ashwin31** [comment]
Hi @inoks Because of the lack of support, no one available to work on heroku deployment.
Can you help with that?

**Comment by inoks** [comment]
@ashwin31 I see some work here already been done by @nikhila05 [here](https://github.com/MicroPyramid/Django-CRM/pull/13). 
Do you what happen with those changes?

**Comment by ashwin31** [comment]
Yes @inoks and it broke for some reason. We decided to host it on regular server for demo purpose rather than heroku. I never saw anyone asking for heroku since then.

I will keep this open and work on it if there are any interests received on this in future.

**Comment by ashwin31** [comment]
@inoks can you send a PR for this ?
